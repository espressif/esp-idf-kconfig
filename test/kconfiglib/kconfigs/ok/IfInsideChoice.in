mainmenu "If block inside choice"

config EXPERIMENTAL
    bool "Experimental features"
    default y

choice DRIVE_IN_USE
    prompt "Choose drive type"

    config WARP_DRIVE
        bool "Warp drive"

    config SUBLIGHT_DRIVE
        bool "Sublight drive"


    if EXPERIMENTAL

        config WORMHOLE_DRIVE
            bool "Experimental drive using wormholes"

    endif

endchoice
