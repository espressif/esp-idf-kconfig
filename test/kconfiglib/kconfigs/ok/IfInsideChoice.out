#
# Automatically generated file. DO NOT EDIT.
# Espressif IoT Development Framework (ESP-IDF)  Project Configuration
#
# default:
CONFIG_EXPERIMENTAL=y
# default:
CONFIG_WARP_DRIVE=y
# default:
# CONFIG_SUBLIGHT_DRIVE is not set
# default:
# CONFIG_WORMHOLE_DRIVE is not set
